lib/typesense/options.rb

```diff
@@ -12,7 +12,6 @@ def self.parse(args, &block)
       opts.on('-r', '--protocol ARG', String) { |protocol| options[:protocol] = protocol }
       opts.on('-a', '--api-key ARG', String) { |api_key| options[:api_key] = api_key }
       opts.on('-c', '--collection-name ARG', String) { |collection| options[:collection_name] = collection }
-      opts.on('--polygons') { options[:polygons] = true }
```

Comment:
Intentional removal?

Reply:
Yes, I moved that option [here](https://github.com/performant-software/core-data-connector/pull/158/files#diff-67b86397f5c4c00d3f48f36e79c428f114b6ccfb3f812a7142596197f5cd50ceR111), since it is only ever used by the `typesense:search:index` rake task.